locales/en.default.schema.json

```diff
@@ -850,44 +850,26 @@
         "variant_picker": {
           "name": "Variant picker",
           "settings": {
-            "options": {
-              "header": "Text-only variants",
-              "picker_type": {
-                "label": "Style",
-                "options__1": {
-                  "label": "Dropdown"
-                },
-                "options__2": {
-                  "label": "Pills"
-                }
+            "picker_type": {
+              "label": "Style",
```

Comment:
It used to be `Type` before and not `style`. Maybe we can switch that back to stay as it was 🤔